spec/custom_delegate_spec.rb

```diff
@@ -127,7 +127,7 @@
       'full_size' => { 'width' => '1024', 'height' => '1024' },
       'cookies' => {
         'initialization_vector' => iv_cookie_value,
-        'sinai_authenticated_3day' => auth_cookie_value
+        "sinai_authenticated_1year" => auth_cookie_value
```

Comment:
For consistency, might be good to stick with the single quotes (and above too) since that's what the rest of the code uses.